### PR TITLE
Feature api-endpoints documentation(SWAGGER)#033

### PR DESCRIPTION
**Pull request by Salim-54**
#### What does this PR do?

- Document API endpoints

#### Description of Task to be completed? 

As a user, I should be able to see
- The documentation of all end points

#### How should this be manually tested?

- Clone the repo

- checkout to branch ``ft-api-endpoints-documentation#033``

- Run `npm install` to install modules

- Run `npm  run start` to start the development server

- In your API testing tool (postman recommended) call api `localhost:3000/api/v1/api-docs to 

#### Any background context you want to provide?

- N/A

#### What are the relevant Pivotal Tracker/trello stories?

- https://trello.com/c/Tj3V2a9q/46-swagger-ui-integration033

#### Screenshots (if appropriate)
Screenshot (286)
Screenshot (288)

#### Questions:

- N/A

### CODE REVIEW

**Review by PrinceNiyonshuti — APPROVED** [review]
Looks good

**Review by PrinceNiyonshuti — APPROVED** [review]
nicely done

**Review by Xldivin — APPROVED** [review]

File: .env [hunk 1]
```diff
@@ -1,7 +1,11 @@
...
+# MONGODB_URL=mongodb+srv://root:password7254@cluster0.qgndm.mongodb.net/myBrand?retryWrites=true&w=majority
+MONGODB_URL=mongodb+srv://salim:password7254@cluster0.henb9.mongodb.net/myFirstDatabase?retryWrites=true&w=majority
 JWT_EXPIRES_IN=90d
-PORT=3000
\ No newline at end of file
+PORT=3000
+
+CLOUDINARY_USER_NAME=salim-atlp-brand
+CLOUDINARY_API_KEY=458542692148515
+CLOUDINARY_API_SECRET=7g0ugXbZjMVdZehREBIyDQGmCS0
```
Comment (thepatrickniyo) [inline .env]: Put `.env` file in `.gitignore` , we don't need push to github

File: .nyc_output/processinfo/9bb5eb35-3b76-4e57-82a4-3c490488c524.json [hunk 2]
```diff
@@ -0,0 +1 @@
...s","C:\\Users\\salim\\3D Objects\\backend\\src\\controllers\\auth.controller.js","C:\\Users\\salim\\3D Objects\\backend\\src\\database\\model\\user.model.js","C:\\Users\\salim\\3D Objects\\backend\\src\\utils\\catchAsync.js","C:\\Users\\salim\\3D Objects\\backend\\src\\routes\\blog.route.js","C:\\Users\\salim\\3D Objects\\backend\\src\\controllers\\blog.controller.js","C:\\Users\\salim\\3D Objects\\backend\\src\\database\\model\\blog.model.js","C:\\Users\\salim\\3D Objects\\backend\\src\\utils\\fileUpload.js","C:\\Users\\salim\\3D Objects\\backend\\src\\utils\\cloud.js","C:\\Users\\salim\\3D Objects\\backend\\src\\routes\\user.route.js","C:\\Users\\salim\\3D Objects\\backend\\src\\controllers\\user.controller.js","C:\\Users\\salim\\3D Objects\\backend\\src\\routes\\comments.route.js","C:\\Users\\salim\\3D Objects\\backend\\src\\controllers\\comments.controller.js","C:\\Users\\salim\\3D Objects\\backend\\src\\database\\model\\comments.model.js","C:\\Users\\salim\\3D Objects\\backend\\src\\routes\\subcriber.route.js","C:\\Users\\salim\\3D Objects\\backend\\src\\controllers\\subscriber.controller.js","C:\\Users\\salim\\3D Objects\\backend\\src\\database\\model\\subscribers.model.js"]}
```
Comment (thepatrickniyo) [inline .nyc_output/processinfo/9bb5eb35-3b76-4e57-82a4-3c490488c524.json]: Put `.`nyc_output` under `.gitignore` no need of pushing it to GitHub as well

File: src/database/model/user.model.js [hunk 3]
```diff
@@ -28,18 +28,18 @@ const userSchema = new mongoose.Schema({
...
+  //   type: String,
+  //   required: [true, 'Please confirm your password'],
+  //   validate: {
+  //     // This only works on CREATE and SAVE!!!
+  //     validator: function(el) {
+  //       return el === this.password;
+  //     },
+  //     message: 'Passwords are not the same!'
+  //   }
+  // }
```
Comment (thepatrickniyo) [inline src/database/model/user.model.js]: Remove the commented source codes for best code quality

**Review by thepatrickniyo — CHANGES_REQUESTED** [review]

**Pull request merged**